diff --git a/src/app/layout.tsx b/src/app/layout.tsx
index 7eb0067..3973c60 100644
--- a/src/app/layout.tsx
+++ b/src/app/layout.tsx
@@ -1,5 +1,5 @@
 import type { Metadata } from "next";
-import { Inter } from "next/font/google";
+import localFont from "next/font/local";
 import "./globals.css";
 import { SidebarProvider } from "@/components/ui/sidebar";
 import { AppSidebar } from "@/components/AppSidebar";
@@ -8,9 +8,8 @@ import { DownloadsProvider } from "@/components/providers/DownloadsProvider";
 import { Toaster } from "@/components/ui/sonner";
 import { ReaderProvider } from "@/components/providers/ReaderProvider";
 
-const inter = Inter({
-  variable: "--font-inter",
-  subsets: ["latin"],
+const inter = localFont({
+  src: "./fonts/Inter.ttf",
 });
 
 export const metadata: Metadata = {
@@ -25,7 +24,7 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en" suppressHydrationWarning>
-      <body className={`${inter.variable} antialiased`}>
+      <body className={`${inter.className} antialiased`}>
         <ThemeProvider
           attribute="class"
           defaultTheme="system"
